tests/frontend/test_ops_gate.py

```diff
@@ -220,3 +227,21 @@ def test_merge_measured_pars():
     # gates that have different p[1] parameters
     with pytest.raises(MergeFailure, match="Don't know how to merge these gates."):
         assert D.merge(G)
+
+
+@pytest.mark.parametrize("gate", [ops.Dgate, ops.Coherent, ops.DisplacedSqueezed])
+@pytest.mark.skipif(not tf_available, reason="Test only works if TF is installed")
+def test_tf_batch_in_gates_previously_supporting_complex(gate):
```

Comment:
Minor, but could we not just use `pytest.importorskip` inside the test function? This would avoid the if statement at the top of the file as well